test/local-storage-emulator.js

```diff
@@ -0,0 +1,49 @@
+import { cryptoRandomObjectId } from 'apify-shared/utilities';
```

Comment:
pls use underscore in file names, not dashes

Reply:
I guess that's my fault. I've slowly started converting files to dashes because that's what I'd seen in all JS projects I'd ever worked with, but it seems that there really is no universal consensus on this, so we may as well keep underscores.

Reply:
I personally prefer the dashes too, more likely I am used to them... But I think it really does not matter if we use dashes or underscores, but it should be definitely consistent in the project so I changed it to underscore.